src/app/contact/page.tsx

```
@@ -19,82 +18,67 @@ export default function ContactPage() {
-              <MapPin className="mt-1 size-4 text-sky-200/76" />
-              <span>{siteConfig.location}</span>
-            </div>
-            <div className="grid grid-cols-[1rem_1fr] gap-3 text-sm leading-6 text-white/62">
-              <Clock3 className="mt-1 size-4 text-sky-200/76" />
-              <span>{siteConfig.availability}</span>
-            </div>
-          </CardContent>
-        </Card>
+      <div className="max-w-2xl">
+        <p className="text-lg leading-9 text-white/65">
+          I'm most interested in practical product engineering work: clear web
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)